go.mod

```diff
@@ -9,12 +9,14 @@ require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/mock v1.6.0
 	github.com/google/go-cmp v0.5.9
+	github.com/gorhill/cronexpr v0.0.0-20180427100037-88b0669f7d75
```

Comment:
[The cronexpr](https://github.com/gorhill/cronexpr) has been archived on Jan 11, 2019.

As far as I know, some users are not allowed to use archived dependencies.
I wonder if we have some alternatives?